src/backends/torch/torchlib.cc

```diff
@@ -1042,6 +1106,17 @@ namespace dd
     torch::Device cpu("cpu");
     _module.eval();
 
+    if (!extract_layer.empty() && !_module.extractable(extract_layer))
+      {
+        std::string els;
+        for (const auto &el : _module.extractable_layers())
+          els += el;
```

Comment:
Maybe add a coma or a space, so that the extractable layers name are not juxtaposed (eg. "extractable_layer1extractable_layer2...")

Reply:
ok done